matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -1336,13 +1384,12 @@ var PhenoTips = (function (PhenoTips) {
 
     _getPatientToBeContactedByCurrentNonAdminUser : function(match)
     {
-        if (!this._accessAtLeastEdit(match.reference) && !this._accessAtLeastEdit(match.matched)) {
-            // user does not have at least edit access to any of the patients in the match: do not allow to contact or mark as user-contacted
+        if (this._isAdmin) {
```

Comment:
what is the logic behind the changes in this method?

Reply:
code optimisation